packages/CollabCodeEdit-Client.package/CCEClientUI.class/instance/noRemoteSessionIsLoaded.st

```diff
@@ -0,0 +1,3 @@
+user interaction
+noRemoteSessionIsLoaded
+	^  self session getStatus == CCESessionStatus noRemoteSession
```

Comment:
Is there a reason to use the identity instead of the normal `=`.

Reply:
This is some very old code indeed:D

Reply:
wait.. it is not. sorry thought it was in CCESession